docs/REST API/rest-api.md

```diff
@@ -9026,8 +9051,71 @@ Returns information about the specified software version.
 }
 ```
 
-## Vulnerabilities
+### Get available App Store apps
+
+Returns the list of App Store (VPP) apps purchased in Apple Business Manager.
+
+`GET /api/v1/fleet/software/app_store_apps/:team_id`
+
+#### Parameters
+
+| Name    | Type | In | Description |
+| ------- | ---- | -- | ----------- |
+| team_id | integer | query | **Required**. The team ID. Lists available VPP software for specified team. |
+
+#### Example
+
+`GET /api/v1/fleet/software/app_store_apps/3`
+
+##### Default response
 
+`Status: 200`
+
+```json
+{
+  "app_store_apps": {
+    {
+      "name": "Xcode",
+      "icon_url": "https://is1-ssl.mzstatic.com/image/thumb/Purple211/v4/f1/65/1e/a4844ccd-486d-455f-bb31-67336fe46b14/AppIcon-1x_U007emarketing-0-7-0-85-220-0.png/512x512bb.jpg",
+      "latest_version": "15.4",
+      "app_store_id": 497799835,
+      "added": true
+    },
+    {
+      "name": "Logic Pro",
+      "icon_url": "https://is1-ssl.mzstatic.com/image/thumb/Purple211/v4/f1/65/1e/a4844ccd-486d-455f-bb31-67336fe46b14/AppIcon-1x_U007emarketing-0-7-0-85-220-0.png/512x512bb.jpg",
+      "latest_version": "2.04",
+      "app_store_id": 634148309,
+      "added": false
```

Comment:
@marko-lisica just wanted to call out that this field will always be false because the backend will only return apps that have not been added to this team (per the note here: https://github.com/fleetdm/fleet/issues/18867 under `Changes > Product`)

if we want to get rid of it, it's a 1 line change in the BE so no biggy :D